packages/components/src/placeholder/style.scss

```diff
@@ -30,6 +30,14 @@
 	.components-base-control__label {
 		font-size: $default-font-size;
 	}
+
+	// Separator.
+	hr {
+		border: 0;
+		min-height: $border-width;
+		min-width: $border-width;
+		background-color: $gray-900;
```

Comment:
I'm thinking this could just be a `border-left: 1px solid $gray-900`, and if we override the inherited `border-bottom` we won't even need to set it to `display: none` in the vertical layout.

Reply:
I recognize the rules here are a bit creative, but I actually wrote them this way in hopes that I could make them work for both vertical and horizontal cases, so that there was a reason to add these to the Placeholder component directly. It technically worked:
![rules](https://user-images.githubusercontent.com/1204802/141256215-e956f79f-4a9e-48bd-9ce9-eadbc8473b2f.gif)

But it is a bit noisy and awkward in the vertical orientation. 

I think for now I'll remove the rules from the placeholder component and keep them only for the navigation block.